blog/_posts/2023-01-31-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,259 @@
+more about the Compiler Academy in [this blog article]({% link blog/_posts/2022-11-02-compiler-academy.md %}).
+
+**We implemented the building blocks of a new tool to detect incompatibilities between programs.** Guaranteeing the
+absence of incompatibilities between library dependencies is the cornerstone of a seamless Scala ecosystem. We have
+implemented a prototype of a new tool to detect incompatibilities between two versions of a program (like
+[MiMa](https://github.com/lightbend/mima), but based on the TASTy representation of programs instead of JVM bytecode).
+Our work is based on [TASTy Query](https://github.com/scalacenter/tasty-query), which will be the basis of tools to
+perform static analysis of Scala programs (it is already used by the Metals debugger).
+
+## Roadmap for 2023 (and Beyond)
+
+It is now time to look forward and present our goals for 2023. Our mission remains the same: improve the experience of
+becoming and being a Scala developer, and help the community to build a rich ecosystem of libraries. Ultimately, our
+work will ensure the longevity of Scala by creating a reliable and resilient ecosystem.
+
+We have identified the priorities for 2023 through our discussions with the community (online or at conferences),
+with our [Advisory Board members](https://scala.epfl.ch/#advisory-board-member-list), and with the
+main organizations that are [behind Scala](https://www.scala-lang.org/community/#whos-behind-scala)
+([LAMP](https://lamp.epfl.ch), [Lightbend](https://lightbend.com), and [VirtusLab](https://virtuslab.com)). We are
+grateful to all of them.
+
+In the following subsections we remind you of our ongoing and recurring projects, and we present our most important goals as
+well as some additional stretch goals that would need more resources.
+
+The roadmap we present here is of course subject to adjustments throughout the year.
+
+### Language, Compiler, Standard Library
+
+In addition to keeping the Scala Improvement Process going, and reducing the number of bugs in the compiler and standard
+library, we will:
+
+- **Make the compiler error messages clearer and more actionable.** The compiler should help developers write correct code
+  instead of “just complaining” about incorrect code. The compiler should also guide you to write maintainable code (ie,
+  it should provide linting features). You can find a list of related issues
+  [here](https://github.com/lampepfl/dotty/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%3Areporting).
+- **Bring the compiler’s suggestions into the IDEs.** Instead of emitting only text output, the compiler
+  will produce structured output that IDEs (Metals and IntelliJ) will be able to read to automatically provide “quick
+  fixes” to the users. This should reduce the implementation (and maintenance) costs on the IDE side.
+- **Unfreeze the Scala standard library.** The standard library has not changed (except for bug fixes or some performance
```

Comment:
This is saying _we will_. Is this for sure? Or is this something we'd _like_ to do? I just don't want to give a false promise here unless we're sure we will do this.

Reply:
This is necessary for many things, hopefully we will find a good solution and implement it!